dot/parachain/network-bridge/receiver.go

```diff
@@ -156,10 +158,9 @@ func (nbr *NetworkBridgeReceiver) Name() parachaintypes.SubSystemName {
 func (nbr *NetworkBridgeReceiver) ProcessActiveLeavesUpdateSignal(
 	signal parachaintypes.ActiveLeavesUpdateSignal) error {
 
-	// TODO: #4207 get the value for majorSyncing for syncing package
 	// majorSyncing means you are 5 blocks behind the tip of the chain and thus more aggressively
 	// download blocks etc to reach the tip of the chain faster.
-	var majorSyncing bool
+	majorSyncing := nbr.sync.IsSynced()
```

Comment:
Shouldn't this be
```suggestion
	majorSyncing := !nbr.sync.IsSynced()
```
?

`SyncService.IsSynced()` returns `true` when the sync service believes it has caught up and `majorSyncing` should be `true` when we are *not* caught up, right?

Reply:
Since this is only used in one place, we could also just call `IsSynced()` in the check below (line 181), without introducing the concept of "major syncing". I.e. (if my assumption above is correct):
```suggestion
	if nbr.sync.IsSynced() {
	// update our view
	...
```

Reply:
makes sense, addressed in latest commit!!